### DISCUSSION TITLE
Add "Hide" option to pull request reviews same as there's one for comments

### DISCUSSION THREAD

**Discussion by Jackenmen** [community/community#10465] [Pull Requests]
Currently, you can hide comments on PRs:
![image](https://user-images.githubusercontent.com/6032823/150703507-2bbf4537-af26-4ecb-a149-02bd89cce408.png)
But you cannot hide pull request reviews:
![image](https://user-images.githubusercontent.com/6032823/150703536-082c4a47-c85a-45ab-9dfd-e819e59c792a.png)

This option would be really nice for hiding spam reviews that don't add anything to the conversation and are just noise. This is how I often use the "Hide" option so it's surprising this option isn't also available for pull request reviews.

Example URL: https://github.com/Cog-Creators/Red-DiscordBot/pull/5476

**Comment by mindctrl** [comment]
It's possible to do this in 2026. Now that we have it, it would be nice to make it easier to resolve/hide comments. The current flow is tedious. Click dropdown, click hide, click dropdown menu, click selection, click "Hide comment" button. Lots of clicking. Hiding/resolving comments on PRs is helpful for both humans and AI agents, for multiple reasons.
